pkg/runner/cluster_k8s.go

```diff
@@ -733,6 +751,12 @@ func (c *ClusterK8sRunner) createTestplanPod(ctx context.Context, podName string
 							MountPropagation: &mountPropagationMode,
 						},
 					},
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							v1.ResourceMemory: resource.MustParse("10Mi"),
```

Comment:
I can't think of any reason to have more than 10Mi for a container that just creates a directory.Looks okay to me.

Reply:
Probably less is fine as well, but then again this is freed as soon as the container completes, so it is just for `scheduling` calculations.